node/generate_website_screenshot/README.md

```diff
@@ -0,0 +1,51 @@
+# 📸 Generate Website Screenshot
+
+A Node Cloud Function for generating website screenshot.
+
+_Example input:_
+
+```json
+{
+    "url": "https://google.com/"
+}
+```
+
+_Example output:_
```

Comment:
Let's please add one more output example, one where success is false and there is error `message`.